### PR TITLE
feat(email-card): add feature card for email sending

### PR DESCRIPTION
**Pull request by raghav3615**
Solve issue #222 kindly review it
feat(email-card): add feature card for email sending

Checklist: 
 I have read and followed the contributing guidelines.
 I have developed the component according to the requirements.
 I have tested the component across different browsers and screen sizes.
 I have documented the component for future reference.
 I have submitted a pull request with appropriate labels and referenced this issue number.

### CODE REVIEW

**Comment by raghav3615** [comment]
@sudhashrestha can you please review and approve this

**Review by hari — CHANGES_REQUESTED** [review]
@raghav3615 Thank you for the PR.

We don't need the logical part for this component so you can get rid of the input and state logic. All the content can be static and the goal is to animate the "sent" state when the component is hovered.

**Comment by raghav3615** [comment]
@hari make some changes. Can you review it now

File: content/docs/card/email-feature-card.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+```json
+module.exports = {
+  theme: {
+    extend: {
+    }
+  }
+}
+```
+
```
Comment (hari) [inline content/docs/card/email-feature-card.mdx:30]: You don't need these two steps, you can delete them @raghav3615

**Review by hari — CHANGES_REQUESTED** [review]

File: content/docs/card/email-feature-card.mdx (hunk 1 above)
Comment (raghav3615) [reply content/docs/card/email-feature-card.mdx]: Done

**Review by hari — APPROVED** [review]

**Pull request merged**